lib/rb_shift/util/variable_resolver.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module RbShift
+  module Util
+    # Supporting module to create instance variables of OpenshiftKind
+    module VariableResolver
+      def get_instance_variable(name:, type:, reload: false, filter: nil)
+        var = instance_variable_get(name)
```

Comment:
In my opinion it would be better to use this variant, because it will always be instance variable and they always starts with @.

```
def get_instance_variable(name:, type:, reload: false, filter: nil)
   var = instance_variable_get("@#{name}".to_sym)
```